applications/dashboard/controllers/class.sessioncontroller.php

```diff
@@ -22,14 +22,16 @@ public function stash() {
         $this->deliveryType(DELIVERY_TYPE_BOOL);
         $this->deliveryMethod(DELIVERY_METHOD_JSON);
 
-        $name = $this->Request->post('Name', '');
-        $value = $this->Request->post('Value', '');
+        if ($this->Request->isAuthenticatedPostBack(true)) {
```

Comment:
Weird diff, but I'm just wrapping the method in an `isAuthenticatedPostBack(true)`.